File: Enlarge_Button_Size_VB_Sample/Form1.vb
Imports Syncfusion.Windows.Forms.Tools

Public Class Form1
    Inherits Form
    Public Sub New()
        InitializeComponent()

        Dim dateTimePickerAdv As New DateTimePickerAdv()
        dateTimePickerAdv.Location = New Point(100, 100)
        dateTimePickerAdv.ShowUpDown = True
        dateTimePickerAdv.Font = New Font("Microsoft Sans Serif", 14.25F, FontStyle.Regular, GraphicsUnit.Point, (CByte(0)))
        Me.Controls.Add(dateTimePickerAdv)
    End Sub
End Class

File: Enlarge_button_size_in_c#_sample/Datetimepickeradv_sample/Form1.cs
using Syncfusion.Windows.Forms.Tools;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Datetimepickeradv_sample
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();

            DateTimePickerAdv dateTimePickerAdv = new DateTimePickerAdv();
            dateTimePickerAdv.Location = new Point(100, 100);
            dateTimePickerAdv.ShowUpDown = true;
            dateTimePickerAdv.Font = new Font("Microsoft Sans Serif", 14.25F, FontStyle.Regular, GraphicsUnit.Point, ((byte)(0)));
            this.Controls.Add(dateTimePickerAdv);
        }
    }
}
